vmdk_plugin/utils/plugin_server/plugin_server_linux.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package plugin_server
```

Comment:
There would be only 2 types of `plugin_server`s i.e. `SockPluginServer` for linux and `NpipePluginServer` for windows. We should be able to service `vmdk`, `photon` and `shared` drivers using these `plugin_server`s by providing the corresponding driver instance to `NewPluginServer(..)`. The `plugin_server`'s job is just to expose the driver APIs over unix sock / windows npipe.